BackEnd/profiles/serializers.py

```diff
@@ -144,6 +155,18 @@ class ProfileOwnerDetailViewSerializer(serializers.ModelSerializer):
     email = serializers.ReadOnlyField(source="person.email")
     regions = RegionSerializer(many=True, read_only=True)
     regions_ukr_display = serializers.SerializerMethodField()
+    banner_image = serializers.ImageField(
+        source="banner.image_path", required=False
+    )
+    logo_image = serializers.ImageField(
+        source="logo.image_path", required=False
+    )
+    banner_approved_image = serializers.ImageField(
+        source="banner_approved.image_path", required=False
+    )
+    logo_approved_image = serializers.ImageField(
+        source="logo_approved.image_path", required=False
+    )
```

Comment:
In publicly (i.e. not admin-only) avaialbe API there must be only `banner` and `logo` fields. Which return approved images.

Reply:
Done
Custom ProfileImage field that contain uuid and path fields created. In publicly API only banner and logo fields are available that return approved or last uploaded (for the profile owner) images.